config_defaults/subtests/docker_cli/run_cgroups.ini

```diff
@@ -3,7 +3,7 @@
 cgroup_path = /sys/fs/cgroup/memory/system.slice/docker
 #: which key is used in this test
 cgroup_key_value = memory.limit_in_bytes
-subsubtests = cpu_positive,cpu_zero,memory_positive,memory_no_cgroup,cpu_none,memory_negative,cpu_overflow
+subsubtests = cpu_positive,cpu_zero,memory_positive,memory_no_cgroup,cpu_none,memory_negative,cpu_overflow,cgroup_parent_invalid_name,cgroup_parent_path,cgroup_parent_path_with_hyphens
```

Comment:
Double ugh. 'docker version' returns 1.9.1, API Version 1.21, on both good and bad builds. Package version returns docker-1.9.1-25.el7.x86_64, but that won't help with Fedora/CentOS. Taking a step back now to let this simmer in my background brain.